govuk-social-providers/login/login.ftl

```diff
@@ -38,13 +38,15 @@
                             <div id="kc-form-options" class="${properties.kcFormOptionsClass!}">
                                 <#if realm.rememberMe && !usernameEditDisabled??>
                                     <div class="checkbox ${properties.kcFormGroupClass}">
-                                        <label class="${properties.kcCheckboxLabelClass}">
+                                        <div class="${properties.kcCheckboxLabelClass}">
                                             <#if login.rememberMe??>
-                                                <input id="rememberMe" name="rememberMe" type="checkbox" tabindex="3" checked> ${msg("rememberMe")}
+                                                <input id="rememberMe" name="rememberMe" type="checkbox" tabindex="3" checked>
+                                                <label for="rememberMe">${msg("rememberMe")}</label>
                                             <#else>
-                                                <input id="rememberMe" name="rememberMe" type="checkbox" tabindex="3"> ${msg("rememberMe")}
+                                                <input id="rememberMe" name="rememberMe" type="checkbox" tabindex="3">
+                                                <label for="rememberMe">${msg("rememberMe")}</label>
                                             </#if>
-                                        </label>
+                                        </div>
```

Comment:
Yeah possibly — if we did that, we’d need to delete the theme.properties entry for `kcCheckboxLabelClass` and add the class in the template instead, as the new GOV.UK Elements checkboxes wrap the whole component in a `<div>` rather than a label.

Probably a good idea to do it — it’s confusing terminology as it stands.

Fixed.